src/subcommands/subcommand.go

```diff
@@ -13,6 +13,8 @@ import (
 	"google.golang.org/grpc"
 )
 
+var defaultIndexPath = path.Join(os.ExpandEnv("$HOME/.semantifly"), "default")
```

Comment:
const because you never modify it and it's a default

Reply:
i tried already to make it a const but it doesn't let me since it says the environment variable can change

Reply:
Ah, I forgot golang consts needed to be evaluated at compile-time. Never mind